### PR TITLE
Onboarding document

### PR DESCRIPTION
**Pull request by namankrs**
Will update remaining after getting EUID and laptop setup done.

### CODE REVIEW

File: ONBOARDING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Welcome aboard
+## Follow these instructions 
+
+-  Get yourself added to "Metrics Dashboard" google chat room
+- Get your TW-associated id to the trello wall "metrics-dashboard"
```
Comment (kumarsi) [inline ONBOARDING.md]: Link to Trello wall

File: ONBOARDING.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- Get added to docker hub thoughtworksindia group
+- Get your TW github Id added to [tw-hygieia](https://github.com/tw-hygieia)
```
Comment (kumarsi) [inline ONBOARDING.md:7]: Give the github teams link

**Review by kumarsi — CHANGES_REQUESTED** [review]
The README links have already been made on master, you may want to rebase once again.

**Comment by namankrs** [comment]
Changes incorporated

**Review by kumarsi — APPROVED** [review]

**Pull request merged**